pandas/core/indexes/base.py

```diff
@@ -2022,18 +2022,48 @@ def hasnans(self):
 
     def isna(self):
         """
-        Detect missing values
+        Detect missing values.
+
+        Return a boolean same-sized object indicating if the values are NA.
+        NA values, such as None or :attr:`numpy.NaN`, get mapped to True
+        values.
+        Everything else get mapped to False values. Characters such as empty
+        strings `''` or :attr:`numpy.inf` are not considered NA values
+        (unless you set :attr:`pandas.options.mode.use_inf_as_na` `= True`).
 
         .. versionadded:: 0.20.0
 
         Returns
         -------
-        a boolean array of whether my values are NA
+        numpy.ndarray
+            A boolean array of whether my values are NA
 
         See also
         --------
-        isnull : alias of isna
+        pandas.Index.isnull : alias of isna
+        pandas.Index.notna : boolean inverse of isna
+        pandas.Index.dropna : omit entries with missing values
         pandas.isna : top-level isna
+
+        Examples
+        --------
+        Show which entries in a pandas.Index are NA. The result is a
```

Comment:
a -> an. array to `numpy.ndarray`.

Reply:
We left it as 'array', because you can see the data type in the returns section (recommended as more user friendly by @jorisvandenbossche)